packages/dbml-core/src/parse/mysql/parser.pegjs

```diff
@@ -456,8 +456,8 @@ ListOfNames = first:name rest:(comma _ name)*
 
 // Comment Syntax: Support "COMMENT 'this is a comment'"
 Comment 
-	= "COMMENT" _ ("=" _)? comment: "'" c:[^']* "'" {return c.join('')}
-	/ "COMMENT" _ ("=" _)? comment: "\"" c:[^"]* "\"" {return c.join('')}
+	= "COMMENT"i _ ("=" _)? comment: "'" c:[^']* "'" {return c.join('')}
+	/ "COMMENT"i _ ("=" _)? comment: "\"" c:[^"]* "\"" {return c.join('')}
```

Comment:
Are there any issues if we ignore the case sensitive of this word? :thinking: Just want to learn it.

Reply:
There is no issue. We forgot to ignore the case sensitive for this token in the past :pray:

Most of the SQL keywords are case-insensitive so it's better if we can parse the keywords correctly.

Reply:
Try to create another Asana task to list the remaining keywords that tie in to these same problems. It could be brought up again for future updates.